### PR TITLE
Add imported NoW submissions to doc d/l modal

### PR DESCRIPTION
**Pull request by BenHarrington**
# Main

- Add imported NoW Submission documents to download modal

# Other

- What other side fixes, tweaks did you add on the side?
- N/A

# How to test

- N/A

# Notes

- Any notes/screenshots for this PR?
- N/A

### CODE REVIEW

File: services/core-api/app/api/now_applications/models/now_application.py [hunk 1]
```diff
@@ -426,7 +428,8 @@ def get_filtered_submissions_documents(cls, now_application):
                     'is_consultation_package': False,
                     'filename': doc.filename,
                     'now_application_id': now_application.now_application_id,
-                    'document_manager_guid': None
+                    'document_manager_guid': None,
+                                                                              # 'update_timestamp': doc.update_timestamp
```
Comment (JustinMacaulay) [inline services/core-api/app/api/now_applications/models/now_application.py]: is this commented out piece needed?

**Review by JustinMacaulay — COMMENTED** [review]
Looks good, just one comment.

File: services/core-api/app/api/now_applications/models/now_application.py (hunk 1 above)
Comment (BenHarrington) [reply services/core-api/app/api/now_applications/models/now_application.py]: Removed.

**Review by JustinMacaulay — COMMENTED** [review]
I am happy with this and happy to approve after you walk Arturo through it

**Review by areyeslo — APPROVED** [review]
Well done Ben!!! Thanks for your fast implementation on this fix.

**Pull request merged**